uberfire-showcase/uberfire-webapp/hs_err_pid1653.log

```diff
@@ -0,0 +1,390 @@
+#
+# There is insufficient memory for the Java Runtime Environment to continue.
+# Native memory allocation (mmap) failed to map 1431306240 bytes for committing reserved memory.
+# Possible reasons:
+#   The system is out of physical RAM or swap space
+#   The process is running with CompressedOops enabled, and the Java Heap may be blocking the growth of the native heap
```

Comment:
This file should be removed!

Reply:
Thanks @pefernan .Sure, I'll remove the file and commit the changes.

Reply:
The file has been deleted and merged with previous commit.